File: examples/functions/rest-api-methods/files/[...path]/get.py
def handler(event, path):
    """GET /files/* — catch-all, path captures everything after /files/"""
    segments = path.split("/") if path else []
    return {
        "status": 200,
        "body": {
            "path": path,
            "segments": segments,
            "depth": len(segments),
        },
    }

File: examples/functions/rest-api-methods/files/[...path]/get.rs
use serde_json::{json, Value};

// GET /files/* — catch-all, path captures everything after /files/
pub fn handler(event: Value) -> Value {
    let path = event["path"].as_str().unwrap_or("");
    let segments: Vec<&str> = if path.is_empty() { vec![] } else { path.split('/').collect() };

    json!({
        "status": 200,
        "headers": { "Content-Type": "application/json" },
        "body": serde_json::to_string(&json!({
            "path": path,
            "segments": segments,
            "depth": segments.len(),
        })).unwrap()
    })
}
